### PR TITLE
All operations work

### PR DESCRIPTION
**Pull request by kozlofski**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+    else if (command == "multiply") {
+        std::string result = std::to_string(first * second);
+        return result;
+    }
+    else if (command == "divide") {
+        std::string result;
+        if (second == 0) {
+            result = "Division by 0";
+        }
+        else {
```
Comment (kuchichan) [inline module1/homework/calculate/calculate.hpp:23]: Możesz pominąć ten else jeśli zwrócisz od razu "Division by 0"

File: module1/homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
 #include <string>
 
-std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+std::string calculate(const std::string &command, int first, int second)
+{
+    if (command == "add") {
+        std::string result = std::to_string(first + second);
+        return result;
```
Comment (DamiXioR) [inline module1/homework/calculate/calculate.hpp:8]: in my opinion you used too many variables, it's enough to use return + action

Comment (kozlofski) [reply module1/homework/calculate/calculate.hpp:8]: Yes, I know :) I had an error here, I was unable to return it in one line, so I added this variable in those all lines, but the error was somewhere else and I didn't simplify it after.

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1429], "points": [2, 5] }

**Pull request closed without merging**